service_msgs/CMakeLists.txt

```diff
@@ -0,0 +1,35 @@
+cmake_minimum_required(VERSION 3.5)
+
+project(service_msgs)
+
+# Default to C++14
+if(NOT CMAKE_CXX_STANDARD)
+  set(CMAKE_CXX_STANDARD 14)
+endif()
+if(CMAKE_COMPILER_IS_GNUCXX OR CMAKE_CXX_COMPILER_ID MATCHES "Clang")
+  add_compile_options(-Wall -Wextra -Wpedantic)
+endif()
+
+find_package(ament_cmake REQUIRED)
+find_package(builtin_interfaces REQUIRED)
+find_package(rosidl_core_generators REQUIRED)
+find_package(unique_identifier_msgs REQUIRED)
+
+set(msg_files
+  "msg/ServiceEventInfo.msg"
+)
+
+rosidl_generate_interfaces(${PROJECT_NAME}
+  ${msg_files}
+  DEPENDENCIES builtin_interfaces unique_identifier_msgs
+  ADD_LINTER_TESTS
+)
+
+if(BUILD_TESTING)
+  find_package(ament_lint_auto REQUIRED)
+  ament_lint_auto_find_test_dependencies()
+endif()
+
+ament_export_dependencies(rosidl_default_runtime)
```

Comment:
Just adding a note here to remind us to update this to `rosidl_core_runtime` after https://github.com/ros2/rosidl_defaults/pull/22 is resolved.

Reply:
This was resolved in [f274555](https://github.com/ros2/rcl_interfaces/pull/143/commits/f27455529315fbbc6f49a16f852156e379bab2c5)

Reply:
and [39827df](https://github.com/ros2/rcl_interfaces/pull/143/commits/39827df73c2bc21ce97e14856a97dc4890a65eab)****